module/applications/actor/group-sheet.mjs

```diff
@@ -0,0 +1,299 @@
+      case "convertCurrency":
+        return Dialog.confirm({
+          title: `${game.i18n.localize("DND5E.CurrencyConvert")}`,
+          content: `<p>${game.i18n.localize("DND5E.CurrencyConvertHint")}</p>`,
+          yes: () => this.actor.convertCurrency()
+        });
+      case "removeMember":
+        const removeMemberId = button.closest("li.group-member").dataset.actorId;
+        return this.object.system.removeMember(removeMemberId);
+      case "movementConfig":
+        const movementConfig = new ActorMovementConfig(this.object);
+        return movementConfig.render(true);
+    }
+  }
+
+  /* -------------------------------------------- */
+
+  /**
+   * Handle clicks to item control buttons on the group sheet.
+   * @param {PointerEvent} event      The initiating click event
+   */
+  #onClickItemControl(event) {
+    event.preventDefault();
+    const button = event.currentTarget;
+    switch ( button.dataset.action ) {
+      case "itemCreate":
+        return this.#createItem(button);
+      case "itemDelete":
+        const deleteLi = event.currentTarget.closest(".item");
+        const deleteItem = this.actor.items.get(deleteLi.dataset.itemId);
+        return deleteItem.deleteDialog();
+      case "itemEdit":
+        const editLi = event.currentTarget.closest(".item");
+        const editItem = this.actor.items.get(editLi.dataset.itemId);
+        return editItem.sheet.render(true);
+    }
+  }
+
+  /* -------------------------------------------- */
+
+  /**
+   * Handle workflows to create a new Item directly within the Group Actor sheet.
+   * @param {HTMLElement} button      The clicked create button
+   * @returns {Item5e}                The created embedded Item
+   */
+  #createItem(button) {
+    const type = button.dataset.type;
+    const system = {...button.dataset};
+    delete system.type;
+    const name = game.i18n.format("DND5E.ItemNew", {type: game.i18n.localize(`DND5E.ItemType${type.capitalize()}`)});
+    const itemData = {name, type, system};
+    return this.actor.createEmbeddedDocuments("Item", [itemData]);
+  }
+
+  /* -------------------------------------------- */
+
+  /**
+   * Handle clicks on member names in the members list.
+   * @param {PointerEvent} event      The initiating click event
+   */
+  #onClickMemberName(event) {
+    event.preventDefault();
+    const member = event.currentTarget.closest("li.group-member");
+    const actor = game.actors.get(member.dataset.actorId);
+    if ( actor ) actor.sheet.render(true, {focus: true});
+  }
+
+  /* -------------------------------------------- */
+
+  /**
+   * Handle clicks on an item name to expand its description
+   * @param {PointerEvent} event      The initiating click event
+   */
+  #onClickItemName(event) {
+    return game.system.applications.actor.ActorSheet5e.prototype._onItemSummary.call(this, event);
+  }
+
+  /* -------------------------------------------- */
+
+  /** @override */
+  async _onDropActor(event, data) {
```

Comment:
Should probably add support for dropping folders of actors, but probably not necessary for initial implementation.

Reply:
I think we need to limit scope. It's part of the reason we have so many un-merged PRs in this repo is because we struggle to make changes small enough that they can be reviewed and merged iteratively. We have to keep everything from becoming a thousand LOC change otherwise we never get anywhere.